api/admin/controller/work_editor.py

```diff
@@ -81,8 +79,8 @@ def details(self, identifier_type, identifier):
         annotator = AdminAnnotator(self.circulation, flask.request.library)
         # Don't cache these OPDS entries - they should update immediately
         # in the admin interface when an admin makes a change.
-        return entry_response(
-            AcquisitionFeed.single_entry(self._db, work, annotator), cache_for=0,
+        return AcquisitionFeed.single_entry(
+            self._db, work, annotator, max_age=0
```

Comment:
Does this need `max_age=0`? I thought `single_entry` sets max_age to 0 by default.

Reply:
Yes, you're right.